draft-irtf-cfrg-aead-limits.md

```diff
@@ -379,6 +379,14 @@ q <= (p^(1/2) * 2^(129/2) - 1) / (L + 1)
 
 ### Integrity Limit
 
+<!--
+    We follow {{AEBounds}} and use the improved integrity bound from
+    {{GCMProofs}}, Equation (22), which gives the term (v * (L+1)) * \delta(...).
+    Assuming s + q + v < 2^64, one can bound \delta(...) <= 1.7 <= 2.
+-->
+
+Assuming `s + q + v < 2^64` implies this bound:
```

Comment:
Can we cite Equation 22 then?  That is:

```suggestion
Applying Equation (22) from {{GCMProofs}}, in which the assumption of
`s + q + v < 2^64` ensures that the delta function cannot produce a value
greater than 2, the following bound applies:
```

That would let us drop the comment entirely and explain the derivation more directly.

Reply:
Agreed, cleaner this way.